.gitignore

```diff
@@ -2,4 +2,4 @@
 __pycache__/
 .cache/
 venv/
-secret.py
\ No newline at end of file
+secret.py
```

Comment:
BTW it'd be better not to include secret.py to gitignore
Since we want to use Heroku secret.py should read `API_KEY` from `os.environ`